source/server/services/user-creation-service.js

```diff
@@ -9,16 +9,35 @@ Space.Object.extend(Space.accounts, 'UserCreationService', {
     Space.messaging.EventPublishing
   ],
 
-  statics: {
-    handleUserCreation(options, user) {
-      // Assign guids as user ids
-      user._id = options.userId;
+  onDependenciesReady() {
```

Comment:
> This is a problem because we can not send an event that user account has been created from the static block when user was created implicitly after users has logged in for the first time using one of the OAuth services.

This is a tricky one for sure. Maybe take a look at https://github.com/splendido/meteor-accounts-meld to see if there's anything of interest